assets/icons/logo.svg

```diff
@@ -1,3 +1,237 @@
-<?xml version="1.0"?>
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 240 240">
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 23.0.3, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 width="500px" height="500px" viewBox="0 0 500 500" style="enable-background:new 0 0 500 500;" xml:space="preserve">
```

Comment:
The width = 500 shows as:

![image](https://user-images.githubusercontent.com/11762685/58909337-b84a4600-86c7-11e9-84da-fbfcc4ada5c1.png)

Do we want to cut the horizontal whitespace down?

Reply:
ah, i see you are already fixing this with: https://github.com/google/docsy/pull/79